Pipfile

```diff
@@ -1,92 +1,79 @@
 [[source]]
-url = "https://pypi.python.org/simple"
+url = "https://pypi.org/simple"
```

Comment:
Question: Did you use a specific command/tool for rearraning the Pipfile and pipfile.lock or do this manually?

Reply:
I had so many incompatibilities with the upgrade, that I ended up deleting the pip and lock files and starting over. So this was generated by the latest version of Pipenv